### PR TITLE
Add laserengineer name 

### PR DESCRIPTION
**Pull request by laserengineer**
#36

### CODE REVIEW

File: Laserengineer-Mudanjiang [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# GitHub for Developers
+#Mudanjiang
```
Comment (nmuennich) [inline Laserengineer-Mudanjiang:3]: What is latitude and longitude of Mudanjiang?

Comment (BorealGitManager) [inline Laserengineer-Mudanjiang:3]: Where is a good restaurant in Mudanjiang? :fish:

**Comment by eandrada** [comment]
ni hao!

File: Laserengineer-Mudanjiang (hunk 1 above)
Comment (eandrada) [reply Laserengineer-Mudanjiang:3]: 44.5517° N, 129.6332° E

Comment (laserengineer) [reply Laserengineer-Mudanjiang:3]: 44.5517° N, 129.6332° E, souther than Edmonton

**Review by nmuennich — APPROVED** [review]
This is perfect.

**Pull request merged**